rts/idris_rts.h

```diff
@@ -112,6 +114,27 @@ struct VM_t {
 
 typedef struct VM_t VM;
 
+
+/* C data interface: allocation on the C heap.
+ *
+ * Although not enforced in code, CData is meant to be opaque
+ * and non-RTS code (such as libraries or C bindings) should
+ * access it only in the following three ways:
+ *
+ *   CData cd = cdata_allocate(...);
+ *     or
+ *   void * ptr = malloc(...);
+ *   CData cd = cdata_manage(ptr, ...);
+ *
```

Comment:
inserting "followed by" here would make this comment more readable. This should also appear on the readthedocs site.